README.rst

```diff
@@ -0,0 +1,91 @@
+############
+edx-learndot
+############
+
+***********
+What is it?
+***********
+
+A Django app for integrating `Open edX`_ and `Learndot`_.
+
+********
+Features
+********
+
+Right now its entire raison d'être is automatically updating Learndot
+enrolment records when a learner passes an edX course.
+
+************
+Installation
+************
+
+Prerequisites
+-------------
+
+* A working Open edX installation.
+
+Installing
+----------
+
+If you're running a Docker devstack, you can clone the repo into your
+``src`` tree and add this to your ``lms`` command in your Docker compose
+file::
+
+  pip install -e /edx/src/edx-learndot
+
+Then you'll need to migrate your database to support storing the
+mappings between Learndot components and edX courses.
+
+*************
+Configuration
+*************
+
+Configuring access to the Learndot API
+--------------------------------------
+
+This enables your edX installation to interact with the `Learndot v2
+API`_.
+
+There are two Django settings you'll need to add:
+
+* Put your Learndot Enterprise base URL (e.g. ``https://your-sandbox.trainingrocket.com/``) in
+  ``settings.LEARNDOT_API_BASE_URL``.
+
+* Put your Learndot API key in ``settings.LEARNDOT_API_KEY``.
```

Comment:
It's worth mentioning that this can be done by adding `LEARNDOT_API_BASE_URL` to the `lms.env.json`, and `LEARNDOT_API_KEY` to lms.auth.json.

Similarly, using ansible variables:
```yaml
EDXAPP_ENV_EXTRA:
     LEARNDOT_API_BASE_URL: https://your-sandbox.trainingrocket.com/
EDXAPP_AUTH_EXTRA:
    LEARNDOT_API_KEY: "your-key-here"
```

Reply:
I don't think it's enough to add them to the JSON files; one of the settings files under `envs/lms` would also have to pull the actual settings from `ENV_TOKENS` or `AUTH_TOKENS`, and that either has to be also noted here, or we'd have to try to get our settings into `edx-platform`, right? Given the multiple ways in which settings can be configured, I was leaving it up to the admin. Do we really want to document all possible options, or assume that if you're adding an app to an edX instance, that you're aware of them and have a preference?